asm/macros/scrcmd.inc

```diff
@@ -415,9 +419,9 @@
     .short \arg0
     .endm
 
-    .macro ScrCmd_04B arg0
+    .macro WaitFanfare sdatID
```

Comment:
**question:** Should we document the differences between Fanfare, Sound, and Music?

Reply:
Sprinkled in some comments, let me know if you think I need to add anything else.